jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java

```diff
@@ -174,18 +174,18 @@ public void unlockAccess() {
 
 	private static void enter(Continuation cont) {
 		try {
+			System.out.printf("Continuation enter cont=%x, state=%x \n", cont, cont.state);
```

Comment:
@LinHu2016 This looks like debug code; it should be removed before the PR is merged?